lib/scala/library-manager/src/main/scala/org/enso/librarymanager/ProjectLoadingFailure.scala

```diff
@@ -1,7 +1,8 @@
 package org.enso.librarymanager
 
-class ProjectLoadingFailure(name: String)
+class ProjectLoadingFailure(name: String, cause: Throwable)
     extends RuntimeException(
       s"The runtime was run in context of a project [$name], " +
-      s"but the project's package could not be loaded."
+      s"but the project's package could not be loaded. ${cause.getMessage}",
```

Comment:
```suggestion
      s"but the project's package could not be loaded (caused by: ${cause.getMessage}).",
```
Maybe worth adding some parentheses or sth to show that the one error is caused by the other and  they are not unrelated.